entries/l/line6.com.json

```
@@ -0,0 +1,11 @@
+{
+  "Line 6": {
+    "domain": "line6.com",
+    "tfa": [
+      "totp"
+    ],
+    "categories": [
+      "other"

```

Looks like the primary use of an account on line6.com is to buy their products. Is this correct?
```suggestion
      "retail"
```

---

That is correct. Thank you for the guidance on which category to use!